apps/style/common.scss

```diff
@@ -270,6 +271,7 @@ html[dir='rtl'] #codeWorkspace {
   .ace_scroller, { // droplet code view
     background-color: rgba(0, 0, 0, 0.1);
   }
+  // #design-mode-container manages this from within the react component
```

Comment:
this is an unusual place for a comment. does it refer to all of #codeWorkspace.readonly and should it be combined with the comment above?